src/Passwordless/Helpers/Extensions/DoubleExtensions.cs

```diff
@@ -0,0 +1,6 @@
+namespace Passwordless.Helpers.Extensions;
+
+internal static class DoubleExtensions
+{
```

Comment:
get rid of it and use `Convert.ToInt32(double value)`?

Reply:
I like the extension function because it allows for the null conditional operator to be used and just inline the whole conversion. It makes the other code very clean and easy to read.

Reply:
Convert.ToInt32 does apply banker's rounding, while casting discards the decimal bits though. Make sure to check whether it's an issue.

Reply:
Good point. I'll use that instead of casting.

Reply:
Yeah, for basic operations like these I'd prefer to inline. If you really want, you can create a `Pipe` extension method:

```csharp
public static TOut Pipe<TIn, TOut>(this TIn input, Func<TIn, TOut> transform) =>
  transform(input);
```

And then just do `myInput?.Pipe(Convert.ToInt32)`

Reply:
I do love me some functional programming.